### ISSUE TITLE
TTS - 2 bugs - editing text breaks everything (video inside)

### ISSUE THREAD

**Issue by cvrle77** [SubtitleEdit/subtitleedit#8981]
I don't think you've fixed this in newer betas, so here it is:

I am not sure exactly what is causing it, but I've made a full video, 15 minutes. At 6:00 you can see how bug manifests, and you can rewind to see what I did before that, so you can repeat it, or get a clue.

I suggest to rewind at 4:18 because there's another bug. This time, I wanted to edit the text without creating an audio, so I just typed what I want in regenerate box, and pressed OK, and got an error.

https://drive.google.com/file/d/1tKLiSMdLO_d5Vk7X6PNrytuonPqRvWFG/view

**Comment by niksedk** [comment]
thx for the info :)
Latest beta is now hopefully better: https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.8/SubtitleEditBeta.zip

**Comment by cvrle77** [comment]
Nope, everything is the same, I got the exact same thing in latest release the moment I am writing this.

**Comment by niksedk** [comment]
OK, please tell me know to re-produce the errors again using latest beta, thx.

**Comment by cvrle77** [comment]
This issue is still ongoing, and I haven't identified a consistent pattern to reproduce it. The latest occurrence appears similar to previous ones: it starts with audio skipping, becoming garbled, and repeating audio when STOP is pressed. Afterward, the app becomes unresponsive, with no option to press anything, and eventually crashes completely. It's incredibly frustrating how much credit has been wasted due to this bug.
